packages/mobile/src/screens/explore-screen/tabs/ForYouTab/ForYouTab.tsx

```diff
@@ -60,12 +63,20 @@ const tiles = [
 export const ForYouTab = () => {
   const styles = useThemedStyles(createStyles)
 
+  const ref = useRef<ScrollView>(null)
+  useScrollToTop(() => {
+    ref.current?.scrollTo({
+      y: 0,
+      animated: true
+    })
+  })
+
   return (
-    <ScrollView style={styles.tabContainer}>
+    <ScrollView style={styles.tabContainer} ref={ref}>
```

Comment:
would it make sense to have special scroll to top flatlist/scrollview components at some point that have useScrollToTop baked in?

Reply:
Yeah I was going back and forth on this. We would need one for flatlist, section list, and scroll view. And then we don't necessarily want every instance to support scroll to top. So i opted for just using the hook in relevant places. But I could be convinced to make custom components for sure